projectatomic/ai-studio
# Environments depends on recipe and model

### What does this PR do?

Index the environments (applications) by (recipeId, modelId) instead of recipeId only.

### What issues does this PR fix or reference?

Fixes #281 

### How to test this PR?

Start, stop, restart applications from environments page and recipe page. Should work as before
